### PR TITLE
[DOC Canary] Added comments for ember testing helper scripts

### PR DESCRIPTION
**Pull request by gautamvij**
added documentation for issue https://github.com/emberjs/ember.js/issues/13862

### CODE REVIEW

File: packages/ember-testing/lib/helpers/click.js [hunk 1]
```diff
@@ -1,3 +1,18 @@
+/**
```
Comment (rwjblue) [inline packages/ember-testing/lib/helpers/click.js]: can you move this comment block down to directly above the `click` function below?

**Comment by rwjblue** [comment]
Can you:

* add the following snippet to the top of each helper file:

```js
/**
@module ember
@submodule ember-testing
*/
```

* Remove the existing documentation in https://github.com/emberjs/ember.js/blob/master/packages/ember-testing/lib/helpers.js that this is replacing.

Thank you!

**Comment by rwjblue** [comment]
Looks great! Would you mind squashing the commits?

**Comment by rwjblue** [comment]
Also will need a rebase

**Comment by mixonic** [comment]
Closed in https://github.com/emberjs/ember.js/pull/14107 - Thanks!

**Pull request closed without merging**